### ISSUE TITLE
macos: not detected as a monospace font

### ISSUE THREAD

**Issue by andypiper** [IdreesInc/Monocraft#26]
In macOS Font Book, there's a smart collection for Monospaced (Fixed Width) fonts.

I've tried to look into how this is determined, and I'm not sure. FontForge docs seems to say that it (FF) should decide for itself when the font is monospaced, although it [also mentions some (OS/2?) setting](https://fontforge.org/docs/faq.html#faq-monospace) for this, and I don't know if this would have an impact.

tl;dr it would be nice if Monocraft was automatically added to the Fixed Width smart collection in Font Book, and I'm not sure why it doesn't show up there.

**Comment by ChiefArug** [comment]
It seems that IntelliJ also does not recognise it as a monospace font.
In Settings -> Editor -> Colour scheme -> Console Font there is an option to only show monospaced fonts, and if that is on Monocraft does not show up.

There is an article on their support page here: https://intellij-support.jetbrains.com/hc/en-us/articles/360000068140 that says it can have something to do with it not including a **bold** version.

**Comment by IdreesInc** [comment]
Problem is due to #77
